### ISSUE TITLE
make operator failed

### ISSUE THREAD

**Issue by Yancey1989** [PaddlePaddle/Paddle#4432]
Execute `make` failed under `/paddle/build/python` folder:
```bash
Scanning dependencies of target operator
[ 94%] Building CXX object paddle/framework/CMakeFiles/operator.dir/operator.cc.o
/paddle/paddle/framework/operator.cc: In member function ‘void paddle::framework::OperatorBase::GenerateTemporaryNames()’:
/paddle/paddle/framework/operator.cc:194:37: error: variable ‘std::atomic<long unsigned int> gUniqId’ has initializer but incomplete type
   static std::atomic<size_t> gUniqId(0UL);
                                     ^
paddle/framework/CMakeFiles/operator.dir/build.make:62: recipe for target 'paddle/framework/CMakeFiles/operator.dir/operator.cc.o' failed
```

**Comment by luotao1** [comment]
duplicated with #4417 , and fixed by #4423 .

**Comment by Yancey1989** [comment]
OK, thanks, I'll close this issue.
